### ISSUE TITLE
VM起動後にharoが起動していない

### ISSUE THREAD

**Issue by altnight** [beproud/beproudbot#171]
### 目的

- VM起動後にbotが動いていてほしい
- `sudo systemctl enable haro` が必要

### 入出力

- /etc/systemd/system/haro.service が起動状態になっている

### 関連機能、影響範囲

特になし

### デモ方法

- VMを落とす -> 起動して haro.service のプロセスが動いていないことを確認

**Comment by altnight** [comment]
https://github.com/beproud/beproudbot/blob/master/deployment/roles/haro/tasks/main.yml#L58
https://docs.ansible.com/ansible/latest/modules/systemd_module.html

`enabled: yes` をいれればok

**Comment by NaoY-2501** [comment]
@altnight 
https://github.com/beproud/beproudbot/blob/master/deployment/roles/haro/tasks/main.yml#L65L68 に `enabled: yes` を追加、ですかね？(更新で当時のリンク先と行数がズレてしまった？
ref. https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html

**Comment by altnight** [comment]
@NaoY-2501 そうですね。現状はworkaroundとして直接VM内で `enabled` にしちゃってますが(`sudo systemctl is-enabled haro`)、ansibleとしてはその行に追加であっています。Slack Event API + Lambda やECS等に移行すると不要になりますが、対応しちゃってもよいと思います。

**Comment by NaoY-2501** [comment]
@altnight こちら対応します

**Comment by NaoY-2501** [comment]
対応のmasterマージdone
